### PR TITLE
feat: remove publish steps from the Makefile

### PR DESCRIPTION
**Pull request by melkir**
## Description of change

This remove the publish parts of the Makefile.
Close https://github.com/shuttle-hq/shuttle/issues/1160

## How has this been tested? (if applicable)

By ensuring that the CI is green.

### CODE REVIEW

**Comment by oddgrd** [comment]
Hey @melkir! :wave: Thanks for this! We can actually remove the publish commands entirely (see updated issue), as they are no longer used.

PS. the failed CI is not your fault :sweat_smile:

**Comment by melkir** [comment]
Thank you for the feedback @oddgrd!

By entirely, do you mean removing these [lines](https://github.com/shuttle-hq/shuttle/pull/1196/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52L272-L302) as well as adapting things attached to it such as this [line](https://github.com/shuttle-hq/shuttle/pull/1196/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52L121) or applying all the changes like on the linked PR?

File: Makefile [hunk 1]
```diff
@@ -118,15 +123,13 @@ DOCKER_COMPOSE_ENV=\
...
+.PHONY: clean images postgres panamax otel deploy test docker-compose.rendered.yml up down shuttle-%
 
 clean:
 	rm .shuttle-*
 	rm docker-compose.rendered.yml
 
-images: shuttle-images postgres panamax otel
-
-shuttle-images: shuttle-auth shuttle-deployer shuttle-gateway shuttle-provisioner shuttle-resource-recorder
+images: shuttle-auth shuttle-deployer shuttle-gateway shuttle-logger shuttle-provisioner shuttle-resource-recorder otel panamax postgres
```
Comment (oddgrd) [inline Makefile:132]: I think we can leave the changes from here and up out for now.

**Review by oddgrd — COMMENTED** [review]
Thanks, that's great! I think we can leave the other changes out for now, and just keep the removal of all the commands..

**Review by oddgrd — APPROVED** [review]
Perfect, thanks!

**Pull request merged**